cmd/podmanV2/utils/alias.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import "github.com/spf13/pflag"
+
+func AliasFlags(f *pflag.FlagSet, name string) pflag.NormalizedName {
```

Comment:
Can you add a doc comment?

Reply:
oops

Reply:
Once we have `golint` enabled again, CI will complain. Such things slip through very easily.